django_celery_results/backends/database.py

```diff
@@ -54,6 +54,53 @@ def exception_safe_to_retry(self, exc):
                 return True
         return False
 
+    def _get_extended_properties(self, request, traceback, using):
```

Comment:
https://github.com/celery/django-celery-results/pull/316/commits/36aeaebff3046713c6923c2d185028b903acae9d

Reply:
Hi @AmitPhulera, I think I was not entirely clear.
My intention was for you to call the `self._get_result_meta` defined by the super class, as it already implements conditionally populating the meta based on `self.app.conf.find_value_for_key('extended', 'result')`

With that meta you can adjust it to the django_celery_results backend specifics like `task_args` instead of `args`, adding `periodic_task_name` if needed and storing argsrepr and kwargsrepr if specified.

Reply:
Hey @AllexVeldman, 
I tried to use `self._get_result_meta()` but I don't think it was of any use. We have to alter/remove many properties and then we have to use `self.app.conf.find_value_for_key('extended', 'result')` anyways to add `peridoic_task_name`.
I felt like we were taking the worst of both worlds it did not simplify anything. But this is just my opinion if you feel that approach would be better then I can push the changes and make it use `self._get_result_meta()`.

Reply:
makes sense, then I prefer your original method name over overriding `_get_result_meta`, as its returned properties are different.

Reply:
Hey @AllexVeldman, 
I have reverted the commits to use `_get_extended_properties `.